ansible/bootstrap.yml

```diff
@@ -11,10 +11,49 @@
       include_tasks: "{{ item }}"
       with_fileglob: "/vagrant/vagrant/conf/ansible/playbooks/prestart/*"
 
+    - name: Verify default variables
+      include_tasks: default_vars.yml
+
+    - name: Verify Consul ACL default policy
+      template: src=/etc/ansible/templates/consul-acl.hcl.j2 dest=/etc/consul.d/config-merge1.hcl
+      when: consul_agent_acl_default_policy == "deny"
+
     - name: Start hashistack
       systemd: name={{item}} state=started
       loop: "{{ daemons }}"
 
+    - name: Verify Consul ACL deny policy - access deny without token
+      uri:
+        url: http://127.0.0.1:8500/v1/acl/tokens
+        status_code: 403
+      register: result
+      until: result.status == 403
+      retries: 5
+      delay: 1
+      when: consul_agent_acl_default_policy == "deny"
+
+    - name: Verify Consul ACL deny policy - access granted with correct token
+      uri:
+        url: http://127.0.0.1:8500/v1/acl/tokens
+        status_code: 200
+        headers:
+          X-Consul-Token: master
```

Comment:
Make this to a variable

Reply:
Do you mean token (value of `X-Consul-Token` header)?

Reply:
Well-- "master" should be replaced with a variable in the playbook with the value of "master"

Reply:
I am thinking about additional functionality, be able to override consul acl master token value, via extra_vars. Do we need it or it does not make sense, due that this vagrant box is not for production usage?

Reply:
Not right now. variables make it easy to change and code clean. DRY

Reply:
done